docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Configure Codacy to block merging pull requests that don't meet your quality standards.
+---
+
+# Integrating Codacy with your Git workflow
+
+Once you've [configured your repository](configuring-your-repository.md) to best match your use case, **integrate Codacy with your Git workflow** to display analysis results and code coverage on your Git provider and block problematic pull requests.
+
+To integrate Codacy with your Git workflow, follow these steps:
+
+1.  [Configuring quality settings](#configuring-quality-settings)
+1.  [Activating the Git provider integration](#git-provider-integration)
+1.  [Blocking merging pull requests](#blocking-pull-requests)
+
+## 1. Configuring quality settings {: id="configuring-quality-settings"}
+
+[Configure the quality settings](../repositories-configure/adjusting-quality-settings.md) of your project to define the gates (minimum acceptable values) and goals for quality metrics such as issues, duplication, complexity, and code coverage.
+
+!!! info "Using code coverage as a quality gate"
+    To use code coverage as a quality gate:
+
+    -   Ensure that code coverage reports are being generated and uploaded to Codacy
+    -   Enable **Diff coverage is under** or **Coverage variation is under** when configuring the quality settings
```

Comment:
A bit long...but all necessary I guess.

I also experimented with the title. Seeing the main point in the title feels like better UX than a generic "info", at least here. Thoughts?

Reply:
In practice it's a good idea :+1:, but I feel like in this particular case we're adding repetition since the title is practically the same as the first sentence in the admonition. I also think the introductory sentence is more "mandatory" to keep things explicit, while the title is more accessory.

We're also losing the "conditional" aspect of this checklist - users only need it if they want to set up a quality gate for coverage.

The pattern that I usually follow is having an admonition where I have a first sentence in bold starting with "If ..." so that users can quickly understand if the information being introduced applies to them or not. You can see some examples of this [here](https://docs.codacy.com/repositories-configure/adjusting-quality-settings/#gates) or [here](https://docs.codacy.com/faq/general/how-can-i-change-or-cancel-my-plan/#if-youre-using-codacy-cloud).

So, taking these suggestions into account, this part could be simplified to:

```suggestion
!!! info
    **If you want to use code coverage as a quality gate:**

    -   Ensure that code coverage reports are being generated and uploaded to Codacy
    -   Enable **Diff coverage is under** or **Coverage variation is under** when configuring the quality settings
```